### ISSUE TITLE
Voice calls disconnecting quite often - beeps

### ISSUE THREAD

**Issue by ganego** [wireapp/wire-android#1405]
## The problem

I do wire phone calls with one contact from time to time, and it quite often happens that the call just disconnects. I then get two beeping sounds (with different pitch) that repeat themselves. This sometimes happens after like 30-40 minutes, but right now I had it happen several times after only ~4 minutes of a call. See the attached log. Both phones use WiFi and both are only a couple of meters away from the WiFi router, so it should not be a connectivity problem.

## Environment

* Wire version that exhibits the issue: 3.4.382
* Last Wire version that did not exhibit the issue (if applicable):
* Android OS version used to run Wire: Android 7.0
* Is this a custom firmware or a stock one: Xiaomi.EU
* Mobile phone model/manufacturer: Xiaomi Redmi Note 4X
* Mobile network type (EDGE/LTE/Wi-Fi/Offline): WiFi

## Details

See above.

## Link to Debug logs

https://gist.github.com/ganego/a670a18e4458576eb8c98d82ea7a1afe

I removed private information like IP addresses and IDs and replaced them (for example `userid_self: xyz` became `userid_self: userid_self_ID`).

**Comment by ivanlechat** [comment]
It's been getting better but I still get this issue

**Comment by ganego** [comment]
I have this now regularly in every call with that one person (I don't have calls with anyone else).
Like every five minutes or so I will hear those beeps and the call pauses (meaning I cannot hear the other person anymore). I have my phone lying next to me and using it with heaphones with integratec mic. When I hear the beeps I have to pick up the phone and press on/off button then it will continue the call most of the time - not always though. I have to repeat this game every couple of minutes.  There should be no energy saving functions active for Wire and the app is always maximized in the foreground.

**Comment by someoneEsle** [comment]
@ganego what phone/os/wire version is the other version on? Can you send us logs right after this happens? (the logs you attached when you created the issue did not contain any useful information, but if you send them again they should be more helpful)

**Comment by ganego** [comment]
Thank you for the answer.  
I'll wait for the release, it would be too complicated having the other person to install it too.  
But I had the chance to do a bit more testing and it seems to be some energy saving function or whatever from my phone (but the options I can access are all disabled for wire). I now had the phone screen on all the time and no problems in a 40 minutes call. Of course not an ideal solution as you can imagine, as the battery usage with screen on is quite high.    
He uses a Moto G4 with whatever Android version for that phone is the newest right now. I'll ask him if he keeps his phone with screen on the next time.  

I'll try it again with disabled screen when 3.8.414 gets released.

**Comment by someoneEsle** [comment]
Thank you so much! Will be waiting for the logs :)

**Comment by someoneEsle** [comment]
Hey @ganego , is this still an issue for you?

**Comment by ganego** [comment]
I did not have longer calls in the past weeks, so I cannot answer this question right now, I'm afraid.

**Comment by someoneEsle** [comment]
Okay, thank you. If it happens again please let us know!

**Comment by ganego** [comment]
Yes still happens after a couple (5 or so) minutes when the screen is off.
